bucket/tageditor.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "3.4.2",
+    "description": "A tag editor with Qt GUI and command-line interface supporting MP4/M4A/AAC (iTunes), ID3, Vorbis, Opus, FLAC and Matroska.",
+    "homepage": "https://github.com/Martchus/tageditor",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
+            "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        }
+    },
+    "shortcuts": [
+        [
+            "tageditor-3.4.2-x86_64-w64-mingw32.exe",
```

Comment:
Having versioned executable in shortcuts in manifest, which has autoupdate is really not good idea...